pysal/esda/smoothing.py

```diff
@@ -550,6 +550,8 @@ def assuncao_rate(e, b):
     s2 = sum(b * ((y - ebi_b) ** 2)) / b_sum
     ebi_a = s2 - ebi_b / (float(b_sum) / len(e))
     ebi_v = ebi_a + ebi_b / b
+    if (ebi_v < 0):
```

Comment:
Will this work since `ebi_v` is an array? I did
`ebi_v = np.where(ebi_v_raw < 0, ebi_b / b, ebi_v_raw)` instead here